src/components/poseWrapper/index.js

```diff
@@ -1,30 +1,64 @@
 import React from "react";
-import * as posenet from '@tensorflow-models/posenet';
+import * as posenet from "@tensorflow-models/posenet";
 import Pose from "../../lib/posenetWrapper";
-import PoseSelector from "../poseSelector"
+import StyledPoseSelector from "../poseSelector";
+import StyledChecklist from "../checklist";
 
 class PoseWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: null,
+      activePose: null
+    };
+    this.handlePose = this.handlePose.bind(this);
```

Comment:
would be great to know what this is doing EXACTLY

Reply:
as in the implemented method handlePose we’re dealing with the component’s state object we need to make sure to bind those methods to this by adding this beautiful line of code to the constructor